packages/editor/src/store/selectors.js

```diff
@@ -1339,6 +1276,28 @@ export function getEditorSettings( state ) {
  * Backward compatibility
  */
 
+/**
+ * Returns state object prior to a specified optimist transaction ID, or `null`
+ * if the transaction corresponding to the given ID cannot be found.
+ */
+export function getStateBeforeOptimisticTransaction() {
```

Comment:
You should add `@deprecated` in jsdoc to be reflected in docs. Same goes for `inSomeHistory`

Reply:
I'm not sure this is a practice we follow for JS code thought? I know that's important for PHP but for js, it doesn't seem to be a widespread practice?

Reply:
Does it have any impacts or api docs generation or something?

Reply:
Yes it does. I checked to make sure before my first comment.